File: XDUploadClient/src/main/kotlin/com/xdupload/XdUpdateBean.kt
package com.xdupload

import java.io.*

/**
 * Created by xingda on 16-8-24.
 */
internal class XdUpdateBean : Serializable {

    var versionCode: Int = 0
    var size: Int = 0
    var versionName: String? = null
    var url: String? = null
    var note: String? = null
    var md5: String? = null
}

File: XdUploadClient/src/main/scala/com/xdandroid/xduploadclient/XdUpdateBean.scala
package com.xdandroid.xduploadclient

import scala.beans.BeanProperty

/**
  * Created by xingda on 16-11-4.
  */
class XdUpdateBean extends Serializable {

  @BeanProperty var versionCode: Int = _
  @BeanProperty var size: Int = _
  @BeanProperty var versionName: String = _
  @BeanProperty var url: String = _
  @BeanProperty var note: String = _
  @BeanProperty var md5: String = _
}
